.circleci/create_circleci_config.py

```diff
@@ -450,13 +450,14 @@ def job_name(self):
         "sudo apt install tesseract-ocr",
         "pip install -U --upgrade-strategy eager pytesseract",
         "pip install -U --upgrade-strategy eager natten",
-        # TODO (ydshieh): Remove this line once `https://github.com/facebookresearch/detectron2/issues/5010` is resolved
-        'pip install -U --upgrade-strategy eager "Pillow<10.0.0"',
+        "pip install -U --upgrade-strategy eager python-Levenshtein",
+        "pip install -U --upgrade-strategy eager opencv-python",
```

Comment:
cc @ydshieh seems like this issue is resolved. I'm adding Nougat here since it relies on Python-Levenshtein and OpenCV. I wonder if there's a way to verify that these tests are actually run